pkg/tlscheck/tlscheck_test.go

```diff
@@ -12,6 +12,48 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func Test_addrValid(t *testing.T) {
+	badAddrs := []string{
+		// should not contain scheme prefix
+		"http://example.com",
+		"tcp://example.com",
```

Comment:
This won't be trimmed by `TrimHTTPPrefixes`. See my other comment.

Reply:
Yes, I added that test case as a documentation for the behavior. There is also one with `tcp://` in `TestTrimHTTPPrefixes` to highlight the same